lib/menu_loader.rb

```diff
@@ -0,0 +1,16 @@
+require 'dish'
+class MenuLoader
+  def initialize
+    @objects = []
+  end
+  def load(file)
+    fail "Data File missing!" if !File.exists? file
+    $/="\n\n"
+    File.open(file, "r").each do |object|
+        @objects << YAML::load(object)
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls. (https://github.com/bbatsov/ruby-style-guide#double-colons)